backend/Server.ts

```diff
@@ -12,4 +14,17 @@ const port = process.env.PORT || 7600;
 
 app.listen(port);
 
-console.log(`Listening on http://localhost:${port}`);
+const fullUrl = `http://localhost:${port}`;
+console.log(`Listening on ${fullUrl}`);
+
+// https://github.com/remy/nodemon/issues/1247
+const pid = process.pid;
+process.on("SIGINT", function () {
+  kill(pid, "SIGTERM");
+  process.exit();
+});
+
+if (process.env.NODE_ENV !== "production" && process.env.FLAX_OPEN === "true") {
```

Comment:
where does `FLAX_OPEN` get set?

Reply:
It doesn't - i put it in the package.json at first but removed it because nodemon restarts the server which was repeatedly reopening the page. I figured I'd leave FLAX_OPEN as an option that can be used manually via `FLAX_OPEN=true pnpm run develop`